### ISSUE TITLE
Bug report: Oups... phpDox encountered a problem and has terminated!

### ISSUE THREAD

**Issue by rodrigobb** [theseer/phpdox#231]
```console
[10.06.2015 - 14:21:27] Saving results to directory '/data/sites/doc/sociaticompletedox/xml'
Oups... phpDox encountered a problem and has terminated!

It most likely means you've found a bug, so please file a report for this and paste the following details and the stacktrace (if given) along:

PHP Version: 5.5.9-1ubuntu4.9 (Linux)
PHPDox Version: 0.7.0
Exception: TheSeer\fDOM\fDOMException (Code: 3)
Location: phar:///[...]/bin/phpdox/fDOMDocument-1.5.0/TheSeer/fDOMDocument/fDOMDocument.php (Line 234)

saving xml file failed

[XML-STRING] [Line: 0 - Column: 0] Fatal Error 6003: output conversion failed due to conv error, bytes 0xA0 0x40 0x70 0x61
[XML-STRING] [Line: 0 - Column: 0] Error 1544: encoder error

#0 phar:///[...]/bin/phpdox/phpdox/Application.php(138): TheSeer\phpDox\Collector\Project->save()
#1 phar:///[...]/bin/phpdox/phpdox/CLI.php(148): TheSeer\phpDox\Application->runCollector()
#2 [...]/bin/phpdox(460): TheSeer\phpDox\CLI->run()
```

Hope it helps

**Comment by theseer** [comment]
Please verify if this also happens with the current release version 0.8.0.

**Comment by rodrigobb** [comment]
You are right. Fixed in 0.8.0

Be aware that the download link given in http://phpdox.de/getting-started.html points to v0.7.0

**Comment by theseer** [comment]
> Be aware that the download link given in http://phpdox.de/getting-started.html points to v0.7.0

Thank you for pointing that out. I'll fix the link asap.
